bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Znunugamma_Monophoton_Ptg40to130_NLO_fxfx/Znunugamma_Monophoton_Ptg40to130_NLO_fxfx_run_card.dat

```diff
@@ -0,0 +1,153 @@
+# ickkw parameter:                                                     *
+#   0: No merging                                                      *
+#   3: FxFx Merging - WARNING! Applies merging only at the hard-event  *
+#      level. After showering an MLM-type merging should be applied as *
+#      well. See http://amcatnlo.cern.ch/FxFx_merging.htm for details. *
+#   4: UNLOPS merging (with pythia8 only). No interface from within    *
+#      MG5_aMC available, but available in Pythia8.                    *
+#  -1: NNLL+NLO jet-veto computation. See arxiv:1412.8408 [hep-ph].    *
+#***********************************************************************
+  3     =  ickkw
+ #**********************************************************************
+#
+#*********************************************************************** 
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+  15.0  =  bwcutoff
+ #**********************************************************************
+# Cuts on the jets                                                     *
+# Jet clustering is performed by FastJet.
+# When matching to a parton shower, these generation cuts should be    *
+# considerably softer than the analysis cuts.                          *
+# (more specific cuts can be specified in SubProcesses/cuts.f)         *
+#***********************************************************************
+  1.0   =  jetalgo    ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+  0.7   =  jetradius  ! The radius parameter for the jet algorithm
+  10.0  =  ptj        ! Min jet transverse momentum
+  -1.0  =  etaj       ! Max jet abs(pseudo-rap) (a value .lt.0 means no c
+#***********************************************************************
+# Cuts on the charged leptons (e+, e-, mu+, mu-, tau+ and tau-)        *
+# (more specific gen cuts can be specified in SubProcesses/cuts.f)     *
+#***********************************************************************
+  0.0   =  ptl      ! Min lepton transverse momentum
+  -1.0  =  etal     ! Max lepton abs(pseudo-rap) (a value .lt.0 means no
+  0.0   =  drll     ! Min distance between opposite sign lepton pairs
+  0.0   =  drll_sf  ! Min distance between opp. sign same-flavor lepton pa
+  0.0   =  mll      ! Min inv. mass of all opposite sign lepton pairs
+  30.0  =  mll_sf   ! Min inv. mass of all opp. sign same-flavor lepton p
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+#***********************************************************************
+  40.0   =  pta     ! Min photon transverse momentum
+  130.0  = ptamax     ! Max photon transverse momentum
```

Comment:
you should use "40.0 = ptgmin" in the run card, and implement the ptgmax cut in the cuts.f, e.g., https://github.com/alpana-hep/genproductions/blob/Alps_dev/bin/GenValidation/updated_validation_cards/dy_fxfx_mll50/dy012jfxfx_ptbinned/Zpt-0To80_cuts_template.f

Reply:
Hi ,
This I tried but I could not add it successfully given I am not at all familiar with fortran. So I need some inputs on this cuts.f file.